rental.rb

```diff
@@ -0,0 +1,13 @@
+class Rental
+  attr_accessor :date, :person, :book
+
+  def initialize(date, person, book)
+    @date = date
+
+    @person = person
+    person.rentals << self
+
+    @book = book
+    book.rentals << self
+  end
+end
```

Comment:
- [ ] I am unable to create a rental as the rental requires a person `object` to rent out to. kindly ensure that the rentals are being created successfully. upon correcting the errors in the `create person` section.
![image](https://user-images.githubusercontent.com/81932478/201354468-d9fddbfd-5d2e-4d49-87f5-bdf92701470b.png)

- [ ] Kindly ensure youre able to:
- list all books
- list all people
- a rentals with a given ID